src/Hosting/Hosting/src/GenericHost/GenericWebHostService.cs

```diff
@@ -73,6 +73,21 @@ public async Task StartAsync(CancellationToken cancellationToken)
                 urls = Options.WebHostOptions.ServerUrls;
             }
 
+            if (string.IsNullOrEmpty(urls))
+            {
+                // HTTP_PORTS and HTTPS_PORTS, these are lower priority than Urls.
```

Comment:
Can we log a warning if we ignore `HTTP_PORTS` and/or `HTTPS_PORTS` becuase `URLS` is set?

Reply:
```
warn: Microsoft.AspNetCore.Hosting.Diagnostics[15]
      Overriding HTTP_PORTS '5002' and HTTPS_PORTS ''. Binding to values defined by URLS instead 'http://*:5003'.
warn: Microsoft.AspNetCore.Server.Kestrel[0]
      Overriding address(es) 'http://*:5003'. Binding to endpoints defined via IConfiguration and/or UseKestrel() instead.
info: Microsoft.Hosting.Lifetime[14]
      Now listening on: http://[::]:5004
```

Reply:
Lots of warnings.......... 😄